architecture/additional_concepts/storage.adoc

```diff
@@ -121,12 +121,43 @@ to policy.
 === Reclaiming
 
 The reclaim policy of a `PersistentVolume` tells the cluster what to do with
-the volume after it is released. Currently, volumes can either be _retained_ or
-_recycled_.
+the volume after it is released. Currently, volumes can either be _Retained_, _Recycled_ or _Deleted_.
 
-Retention allows for manual reclamation of the resource. For those volume
-plug-ins that support it, recycling performs a basic scrub on the volume (e.g.,
-`rm -rf /<volume>/*`) and makes it available again for a new claim.
+Retention allows for manual reclamation of the resource.  For those volume plugins that support it, deletion removes both the `PersistentVolume` object from Kubernetes and deletes associated storage asset in external infrastructure such as AWS EBS, GCE PD or Cinder volume.  Volumes that were dynamically provisioned are always deleted.
+
+[[recycling]]
+
+==== Recycling
+
+If supported by appropriate volume plugin, recycling performs a basic scrub (`rm -rf /thevolume/*`) on the volume and makes it available again for a new claim.
+
+However, an administrator can configure a custom recycler pod template using the Kubernetes controller manager command line arguments as described in xref:../../install_config/master_node_configuration.adoc#master-config-admission-control-config[ControllerArguments]. The custom recycler pod template must contain a `volumes` specification, as shown in the example below:
+
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pv-recycler-
+  namespace: default
+spec:
+  restartPolicy: Never
+  volumes:
+  - name: vol
+    hostPath:
+      path: /any/path/it/will/be/replaced
+  containers:
+  - name: pv-recycler
+    image: "gcr.io/google_containers/busybox"
+    command: ["/bin/sh", "-c", "test -e /scrub && rm -rf /scrub/..?* /scrub/.[!.]* /scrub/*  && test -z \"$(ls -A /scrub)\" || exit 1"]
+    volumeMounts:
+    - name: vol
+      mountPath: /scrub
+----
+====
+
+However, the particular path specified in the custom recycler pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
```

Comment:
can you add some red bullet with "(1)" inside to the yaml so it's clearer what path do you mean here? Something like https://docs.openshift.org/latest/admin_guide/managing_networking.html#admin-guide-deploying-an-egress-router-pod